v3/docs/07-advanced/h-ss58/index.mdx

```diff
@@ -7,28 +7,128 @@ category: advanced
+| 6     | 1    | 4           | 1        |
+| 7     | 1    | 4           | 2        |
+| 8     | 1    | 4           | 3        |
+| 9     | 1    | 4           | 4        |
+| 10    | 1    | 8           | 1        |
+| 11    | 1    | 8           | 2        |
+| 12    | 1    | 8           | 3        |
+| 13    | 1    | 8           | 4        |
+| 14    | 1    | 8           | 5        |
+| 15    | 1    | 8           | 6        |
+| 16    | 1    | 8           | 7        |
+| 17    | 1    | 8           | 8        |
+| 34    | 1    | 32          | 2        |
+
+## Checksum types
+
+Several potential checksum strategies exist within Substrate, giving different length and longevity guarantees.
+There are two types of checksum preimage (known as SS58 and AccountID) and many different checksum lengths (1 to 8 bytes).
+
+In all cases for Substrate, the [Blake2-256](https://en.wikipedia.org/wiki/BLAKE_(hash_function)) hash function is used.
+The variants simply select the preimage used as the input to the hash function and the number of bytes taken from its output.
+
+The bytes used are always the left most bytes.
+The input to be used is the non-checksum portion of the SS58 byte series used as input to the base-58 function, i.e. `concat( <address-type>, <address> )`.
+A context prefix of `0x53533538505245`, (the string `SS58PRE`) is prepended to the input to give the final hashing preimage.
+
+The advantage of using more checksum bytes is simply that more bytes provide a greater degree of protection against input errors and
+index alteration at the cost of widening the textual address by an extra few characters.
+For the account ID form, this is insignificant and therefore no 1-byte alternative is provided.
+For the shorter account-index formats, the extra byte represents a far greater portion of the final address and
+so it is left for further up the stack (though not necessarily the user themself) to determine the best tradeoff for their purposes.
+
+## Simple/full address types and account/address/network identifiers
+
+[The registry](https://github.com/paritytech/ss58-registry) expresses the status of the account/address/network *identifiers*.
+
+*Identifiers* up to value 64 may be expressed in a *simple* format address,
+in which the LSB byte of the identifier value is expressed as the first byte of the encoded address.
+
+For identifiers of between 64 and 16,383, the *full* format address must be used.
+
+The encoding of this is slightly fiddly since we encode as LE,
+yet the first two bits (which should encode 64s and 128s) are already used up with the necessary `01` prefix.
+We treat the first two bytes as a 16 bit sequence,
+and we disregard the first two bits of that (since they're already fixed to be `01`.
+With the remaining 14 bits, we encode our identifier value as little endian,
+with the assumption that the two missing higher order bits are zero.
+This effectively spreads the low-order byte across the boundary between the two bytes.
```

Comment:
```suggestion
This encoding is slightly fiddly since we encode as little endian (LE), yet the first two bits (which should encode 64s and 128s) are already used up with the necessary `01` prefix.
We treat the first two bytes as a 16 bit sequence, and we disregard the first two bits of that (since they're already fixed to be `01`). 
With the remaining 14 bits, we encode our identifier value as LE, with the assumption that the two missing higher order bits are zero.
This effectively spreads the low-order byte across the boundary between the two bytes.
```
Not clear what the "s" is meant to signify here:
> which should encode 64s and 128s